target/zephyr/Kconfig

```diff
@@ -149,6 +149,82 @@ if MENDER_MCU_CLIENT
                 the bootloader configuration, e.g. `sysbuild.conf` if you're using sysbuild.
     endmenu
 
+    menu "Mender Artifact configuration"
+
+        config MENDER_ARTIFACT_GENERATE
+            bool "Generate a Mender Artifact from the built Zephy kernel image"
+            default y
+            help
+                Auto-generates a Mender Artiafact from the Zephyr kernel. It requires the mender-artifact CLI tool
+                installed in the workstation.
+
+        if MENDER_ARTIFACT_GENERATE
+
+            config MENDER_ARTIFACT_NAME
+                string "Name of the artifact"
+                help
+                    Name of the artifact. Mandatory.
+
+            config MENDER_ARTIFACT_TYPE
+                string "Type of payload"
+                default "zephyr-image" if MENDER_ZEPHYR_IMAGE_UPDATE_MODULE
+                help
+                    This is the same as the name of the update module. Mandatory.
+
+            config MENDER_DEVICE_TYPES_COMPATIBLE
+                string "Type of device(s) supported by the Artifact"
+                default MENDER_DEVICE_TYPE
+                help
+                    It can specified more than one device types separated by spaces. Mandatory.
+
+            config MENDER_ARTIFACT_PROVIDES
+                string "Generic KEY:VALUE which is added to the type-info -> artifact_provides section."
```

Comment:
Should we add *space-separated*?

Reply:
Do you mean in all of them (device types, provides and depends)? I think that being in the `help` part is enough.

Reply:
Yup, I missed that. Sorry for the noise!